src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -167,7 +170,8 @@
                                     VerticalAlignment="Center" Click="ButtonSave_Click" 
                                     Style="{StaticResource ToolWindowButtonStyle}"  ToolTipService.ShowOnDisabled="True"
                                     Content="{x:Static components:Resources.ConnectionManager_Save}" 
-                                    ToolTip="{Binding Path=SaveButtonTooltip}">
+                                    ToolTip="{Binding Path=SaveButtonTooltip}"
+                                    AutomationProperties.Name="{Binding Path=SaveButtonTooltip}">
```

Comment:
Inspect sees what is expected:

![image](https://cloud.githubusercontent.com/assets/8933858/26600064/4b41654a-4540-11e7-91dc-e2d8f6be0e39.png)

What do you mean by "control is opened"?